src/data_forwarding/amazon_sqs/forwarder.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import annotations
+
+import logging
+from typing import Any
+from uuid import uuid4
+
+import boto3
+import orjson
+from botocore.client import Config
+from botocore.exceptions import ClientError, ParamValidationError
+
+from data_forwarding.base import BaseDataForwarder
+from sentry.integrations.models.data_forwarder_project import DataForwarderProject
+from sentry.integrations.types import DataForwarderProviderSlug
+from sentry.services.eventstore.models import Event
+
+logger = logging.getLogger(__name__)
+
+DESCRIPTION = """
+Send Sentry events to Amazon SQS.
+
+This integration allows you to forward Sentry events to an Amazon SQS queue for further processing.
+
+Amazon SQS is a fully managed message queuing service that enables you to decouple and scale microservices.
+"""
+
+
+class AmazonSQSForwarder(BaseDataForwarder):
+    provider = DataForwarderProviderSlug.SQS
+    rate_limit = None
+    description = DESCRIPTION
+
+    @classmethod
+    def get_event_payload(cls, event: Event) -> dict[str, Any]:
+        return dict(event.data)
```

Comment:
if it's an Event object we're operating on, I think we can use the [Event's as_dict func](https://github.com/getsentry/sentry/blob/crl/add-new-issues-types-to-webhooks/src/sentry/services/eventstore/models.py#L492)